listenbrainz/webserver/static/js/src/recommendations/Recommendations.tsx

```diff
@@ -0,0 +1,317 @@
+/* eslint-disable jsx-a11y/anchor-is-valid,camelcase */
+
+import { AlertList } from "react-bs-notifier";
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import * as _ from "lodash";
```

Comment:
```suggestion
import {isEqual} from "lodash";
```

And further down in the same file remove the `_.` :
`_.isEqual(…`-> `isEqual(…`